src/MemoryTracing/include/MemoryTracing/SystemMemoryInfoProducer.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MEMORY_TRACING_SYSTEM_MEMORY_INFO_PRODUCER_H_
+#define MEMORY_TRACING_SYSTEM_MEMORY_INFO_PRODUCER_H_
+
+#include "GrpcProtos/Constants.h"
+#include "MemoryTracing/MemoryInfoProducer.h"
+
+namespace orbit_memory_tracing {
+
+// This class periodically produces the SystemMemoryUsage information retrieved from /proc/meminfo
```

Comment:
```suggestion
// This class periodically produces the `SystemMemoryUsage` information retrieved from /proc/meminfo
```

Reply:
Thanks for the suggestion! Now its at a different location than before, but I have updated the comment accordingly.